src/Input/NumberInput.tsx

```diff
@@ -0,0 +1,97 @@
+import React from "react";
+import styled from "styled-components";
+
+import { colors } from "../shared/constants";
+
+const StyledContainer = styled.div`
+  width: 100%;
+  position: relative;
+  overflow-x: hidden;
+`;
+
+const StyledInner = styled.div`
+  width: 100%;
+  display: inline;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+`;
+
+const StyledTitle = styled.div`
+  float: left;
+  font-size: 18px;
+  font-weight: 600;
+  color: ${colors.DARK_GRAY};
```

Comment:
What happens when `title` is super long? It will likely break the layout. Let's make it so that if it is too long for the width, have it cut off with ellipsis.
https://www.w3schools.com/cssref/css3_pr_text-overflow.asp
https://stackoverflow.com/a/17783233

Reply:
Updated
![image](https://user-images.githubusercontent.com/10282189/77673166-341bc480-6f60-11ea-9283-3ffd34671c09.png)